terraform/environments/electronic-monitoring-data/modules/lambdas/main.tf

```diff
@@ -120,18 +120,32 @@ resource "aws_cloudwatch_log_group" "lambda_cloudwatch_group" {
   kms_key_id        = aws_kms_key.lambda_env_key.arn
 }
 
+data "external" "latest_image_update_log_table" {
+  for_each = var.is_image ? { image = 1 } : {}  # Use empty map if not fetching image
+
+  program = ["bash", "${path.root}/bash_scripts/get_latest_image.sh", var.ecr_repo_name, var.function_name]
+  query = {
+    latest_image_uri = ""
+  }
+}
+
 
 resource "aws_lambda_function" "this" {
-  #checkov:skip=CKV_AWS_272:Lambda needs code-signing, see ELM-1975
```

Comment:
Are you sure you want to remove this checkov ignore?

Reply:
good point